use-case-guides/concur-expense-report-lookup/README.md

```diff
@@ -0,0 +1,143 @@
+> 
+
+# Steps
+
+## Step 1: Build in Creator Studio
+
+### Setup use case
+
+1. Start in the Queries Workspace and create a new query.
+2. Provide the Basic Info so the Next Gen Copilot knows how to use this plugin:
+    1. **Query Label**: `Lookup Expenses from Concur` 
+    2. **Short Description:** `Lookup your Expense Reports from Concur. This includes looking up any information regarding the name, status, approver, total to be paid, and more.`
+
+### Setup the API
+
+1. Select the Concur connector that you set up earlier in the [Authentication Guide](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=concur)
+    
+    ![Untitled](Use%20Case%20Tutorial%20Lookup%20Expense%20Reports%20in%20Concur%200ad48c0ab26047b1bab45a82557a0bda/Untitled%202.png)
+    
+2. Define your API action for querying the Expense Reports data from Concur:
+    - Path: `/api/v3.0/expense/reports`
+    - Method: `GET`
+    - Headers:
+        
+        
+        | KEY | VALUE |
+        | --- | --- |
+        | Accept | application/json |
+    - Query parameters:
+        
+        
+        | KEY | VALUE |
+        | --- | --- |
+        | user | {{user.email_addr}} |
+        | limit | 10 |
+        - Selecting `user.email_addr` would ensure that the bot picks up the the email address that is currently mapped to the Bot for the user, and use that as the query parameter in the above API.
+3. Test your setup in Creator Studio and look for a successful execution. A 200 Response Code represents that the API was successfully executed.
+If you check the `OwnerLoginID` or `OwnerName` from the returned API response, you will see that it contains your Email ID and Name respectively. 
+    
+    ![Untitled](Use%20Case%20Tutorial%20Lookup%20Expense%20Reports%20in%20Concur%200ad48c0ab26047b1bab45a82557a0bda/Untitled%203.png)
+    
+
+### Label the API Response
+
+1. Select the `Name` as the Identifier.
+2. Select any of the fields as the Description. In this example, we have selected `Approval Status Name` as the Description.
+3. From among the Additional Fields, select the fields that you want users to query on and label them accordingly.
+    
+    ![Untitled](Use%20Case%20Tutorial%20Lookup%20Expense%20Reports%20in%20Concur%200ad48c0ab26047b1bab45a82557a0bda/Untitled%204.png)
+    
+4. No follow-up action needed.
+
+### Build your Generative Intent
+
+1. Since we want to capture the email ID of the User who is using the Bot, which is further used to filter the Expense Reports API, we will need to provide accurate examples when to trigger this Use Case. Creator Studio will automatically generate utterances which will trigger the Use Case to execute the API and fetch details for your expense reports.
+    
+    ![Untitled](Use%20Case%20Tutorial%20Lookup%20Expense%20Reports%20in%20Concur%200ad48c0ab26047b1bab45a82557a0bda/Untitled%205.png)
```

Comment:
I would fix the generative intents in the image, maybe put the ones you configured below it as bullets